src/content/basics/networkpools/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Network Pools
+description: A general description of network pools as a concept, it's benefits, and some details you should be aware of.
+date: 2021-01-09
+weight: 130
+type: page
+categories: ["basics"]
+user_questions:
+  - What is a network pool?
+  - What are network pools?
+  - In which cloud environments are network pools supported?
+  - Which releases introduced network pools?
+---
+
+# Network Pools
+
+## Definition
+
+A NetworkPool defines a CIDR range for the Control Plane and MachineDeployment nodes of a Cluster.
+
+## Advantages
+
+The main advantages of using NetworkPools are:
+
+- Extending installation default CIDR range.
+- Limit the CIDR range a single cluster can have.
+
+## Lifecycle
+
+NetworkPools must be created before the cluster is created via Management Cluster Kubernetes API.
+
+NetworkPools can be modified as long as the CIDR doesn't overlap with other NetworkPools and contains the original CIDR. For example 10.0.0.0/16 could be modified to 10.0.0.0/15 but not to 10.0.0.0/17.
+
+## Assigning clusters to NetworkPools {#assigning-networkpool}
```

Comment:
What is `{#assigning-networkpool}` for? Should that render in markdown?

Reply:
I checked on nodepools and I think this is used to link directly to this paragraph, but we are not using it I will remove it